h/static/scripts/controllers/index.js

```diff
@@ -11,10 +11,8 @@
 import { CharacterLimitController } from './character-limit-controller';
 import { ConfirmSubmitController } from './confirm-submit-controller';
 import { CopyButtonController } from './copy-button-controller';
-import { DisableOnSubmitController } from './disable-on-submit-controller';
 import { DropdownMenuController } from './dropdown-menu-controller';
 import { FormCancelController } from './form-cancel-controller';
-import { FormController } from './form-controller';
```

Comment:
This supported server-rendered forms which could be refreshed and submitted by JS. This is no longer used. The accounts forms are just regular forms. If we do move to client-side submission for them, we'll probably just use JSON.